src/frame/stream_id.rs

```diff
@@ -10,8 +20,10 @@ pub struct StreamIdOverflow;
 const STREAM_ID_MASK: u32 = 1 << 31;
 
 impl StreamId {
+    /// Stream ID 0.
     pub const ZERO: StreamId = StreamId(0);
 
+    /// The maximum allowed stream ID.
     pub const MAX: StreamId = StreamId(u32::MAX >> 1);
 
     /// Parse the stream ID
```

Comment:
I'm not sure if this function should be publicly exposed or not (perhaps it should be `pub(crate)` instead?)